pkg/apis/ocibuilder/v1alpha1/types.go

```diff
@@ -182,29 +183,25 @@ type AnsibleGalaxy struct {
 type BuildStep struct {
 	// Metadata about the build step.
 	*Metadata `json:"metadata,inline" protobuf:"bytes,1,name=metadata"`
-	// Type of the build framework.
-	// Defaults to docker
-	// +optional
-	Daemon Daemon `json:"daemon,omitempty" protobuf:"bytes,2,opt,name=daemon"`
 	// Stages of the build
```

Comment:
were client-gen out files pushed after removing Daemon field? I don't see'em

Reply:
update-codegen.sh isn't adding any changes - let me double check

Reply:
Were there any changes?

Reply:
Nope - nothing was added apart from an unused line in register.go